zap/src/main/java/org/parosproxy/paros/core/scanner/HostProcess.java

```diff
@@ -499,6 +502,27 @@ protected boolean nodeInScope(String nodeName) {
         return parentScanner.isInScope(nodeName);
     }
 
+    private FilterResult filterNode(StructuralNode node) {
```

Comment:
This can now return boolean, since the result is no longer needed by `canScanNode`.

Reply:
hmm. sure will do that.

Reply:
done.